cypress/e2e/app.cy.ts

```diff
@@ -158,9 +158,8 @@ describe('/mock', () => {
     cy.selectExplorerNode('fourD');
 
     cy.findByRole('figure', { name: 'fourD' }).as('vis').should('be.visible');
-    cy.findAllByRole('slider', { name: 'Dimension slider' })
-      .should('have.length', 2)
-      .last()
+    cy.findAllByRole('slider', { name: /D[0-9]/u }).should('have.length', 2);
+    cy.findAllByRole('slider', { name: 'D1' })
```

Comment:
Gave the domain slider thumbs more specific names so it's easier to find the one we want